src/dialog/validator.js

```diff
@@ -0,0 +1,41 @@
+import _ from 'lodash'
+
+exports.validateFlowSchema = flow => {
+  const errorPrefix = `[Flow] Invalid flow "${flow && flow.location}"`
+
+  if (!flow || !_.isObjectLike(flow)) {
+    return 'Invalid JSON flow schema'
+  }
+
+  if (!flow.version || !_.isString(flow.version)) {
+    return `${errorPrefix}, expected valid version but found none`
+  }
+
+  if (!flow.version.startsWith('0.')) {
```

Comment:
Why this special case?

Reply:
Meaning this current version of Botpress only understands `0.x` flows